install-steps.md

```diff
@@ -1,193 +1,821 @@
-systemctl start libvirtd
-systemctl enable libvirtd
-
-cat << EOF > default_storage.xml 
-<pool type='dir'>
-  <name>default</name>
-  <source>
-  </source>
-  <target>
-    <path>/var/lib/libvirt/images</path>
-    <permissions>
-      <mode>0711</mode>
-      <owner>0</owner>
-      <group>0</group>
-      <label>system_u:object_r:virt_image_t:s0</label>
-    </permissions>
-  </target>
-</pool>
-EOF
-
-virsh pool-define default_storage.xml 
-virsh pool-start default
-virsh pool-autostart default
-
-nmcli connection add ifname provisioning type bridge con-name provisioning
-nmcli con add type bridge-slave ifname eno1 master provisioning
-nmcli connection add ifname baremetal type bridge con-name baremetal
-nmcli con add type bridge-slave ifname eno2 master baremetal
-
-#optional - verify you see the connections 
-nmcli con show
-
-#warning - if you are on an ssh terminal, you will kill connection running this
-nmcli con down eno2
-
-#warning - if you used dhclient to get an IP on eno2, kill the dhclient process
-ps -ef | grep dhclient
-kill -9 <id_of_dhclient_process>
-
-dhclient baremetal
-
-nmcli connection modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method manual
-nmcli con down provisioning
-nmcli con up provisioning 
-
-cat << EOF > provisioning.xml
-<network>
-  <name>provisioning</name>
-  <bridge name='provisioning'/>
-  <forward mode='bridge'/>
-</network>
-EOF
-
-virsh net-define provisioning.xml
-virsh net-start provisioning
-virsh net-autostart provisioning
-
-cat << EOF > baremetal.xml
-<network>
-  <name>baremetal</name>
-  <bridge name='baremetal'/>
-  <forward mode='bridge'/>
-</network>
-EOF
-
-virsh net-define baremetal.xml
-virsh net-start baremetal
-virsh net-autostart baremetal
-```
-
-     # optional - check the networks on virsh
-     virsh net-list
-     Name                 State      Autostart     Persistent
-     ----------------------------------------------------------
-     baremetal            active     yes           yes
-     default              active     yes           yes
-     provisioning         active     yes           yes
-
-```bash
-systemctl restart libvirtd
-
-# still need this? i cant remember why. 
-echo 'ZONE=libvirt' >> /etc/sysconfig/network-scripts/ifcfg-provisioning
-
-#Lets create a user (kni) so we aren't deploying as root and set ssh key
-echo "kni ALL=(root) NOPASSWD:ALL" | tee -a /etc/sudoers.d/kni
-chmod 0440 /etc/sudoers.d/kni
-su - kni -c "ssh-keygen -t rsa -f /home/kni/.ssh/id_rsa -N ''"
-
-#lets make sure python3 is default
-alternatives --set python /usr/bin/python3
-
-
-#Get your pull-secret file on the system
+# Introduction
+
+This write-up will guide you through the process of deploying a Baremetal IPI installation of Red Hat OpenShift 4.
+# Prerequisites
+
+* 6 Physical servers (1 provision node, 3 master and 2 worker nodes)
+* Each server needs 2 NICs pre-configured. NIC1 for the private network and NIC2 for the external network. 
```

Comment:
It's also necessary for the 3 master hosts to have similar enough hardware that the interface names for the NICs are identical. See https://github.com/openshift/installer/issues/2762

Reply:
I'll make sure to make a note of that in prereqs.